tools/finish_release.py

```diff
@@ -224,7 +241,7 @@ def main(args):
         if parsed_args.publish_snaps:
             promote_snaps(version)
         if parsed_args.publish_windows:
-            create_github_release(github_access_token, tempdir, version)
+            create_github_release(github_access_token, tempdir, version, css)
```

Comment:
Sorry I didn't identify this earlier but I think we should do this code signing earlier in the execution of this script. My reasoning is that it's the only part of the script that prompts for user interaction and I expect it to be the most finicky where it'll fail for some reason and we'll have to run the script again (e.g. not on the network to find the CSS, SSH key not inserted, etc.).

To do this, I suggest we add another `if parsed_args.publish_windows:` section and sign the installer there before the `if parsed_args.publish_snaps:` section. I still think we create the GitHub release to help with testing as I described at https://github.com/certbot/certbot/pull/9142. What do you think?

Reply:
If I understand the process here in python, maybe keep it to the one condition check and I can just wrap the installer signing in it's own function and process that before `create_github_release` executes? Let me know if that is more reasonable.

Reply:
I personally like my version better because I expect code signing to be the most finicky, processing the snaps is slow, and I'd like to do the GitHub release last since it's expected to fail during testing. Do you have a problem with code that looks something like:
```
if parsed_args.publish_windows:
    sign_windows_installer(tempdir, css)
if parsed_args.publish_snaps:
    promote_snaps(version)
if parsed_args.publish_windows:
    create_github_release(github_access_token, tempdir, version)
```

Reply:
That looks good, I will try that version out.